vivo_queries/queries/make_academic_article.py

```diff
@@ -23,13 +25,15 @@ def fill_params(connection, **params):
         params['Article'].final_check(year_id)
         params['year'] = year_id
 
-    #make sure none of the n numbers generated before inserting triples have repeating n numbers
+    # make sure none of the n numbers generated before inserting triples have repeating n numbers
     params['Article'].final_check(relationship_id)
-    params['Journal'].final_check(relationship_id)
-    params['Article'].final_check(params['Journal'].n_number)
+    if params['Journal'] is not None:
```

Comment:
I added this when I tried creating a Supported Article for Grants which did not have any Journal information. I am not sure if an article will always be associated with a Journal. Let me know if any changes need to be made for that. Also, I can remove it for the time being if you want as the csv file does not contain field for this.

Reply:
You don't need to specify it. If there is no journal n number, it will still pass the final check and there will be nothing about journals in the triples. Once you get rid of this line I can approve the pull.